src/components/Dashboard/dash-tabs-styles.postcss

```diff
@@ -5,13 +5,9 @@
 
   .tab {
     display: inline-block;
-    width: 49%;
-    margin-right: 1%;
-
-    @media screen and (min-width: $screen-m) {
-      width: 50%;
-      margin-right: 0;
-    }
+    width: 50%;
+    background-color: #ddd;
+    background-image: url('./../../images/legendpattern.jpg');
```

Comment:
Would suggest renaming the 'legendpattern' to something more generic if it's being used across multiple parts. Maybe cardboardpattern?